packages/components/psammead-play-button/src/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { string, oneOf, func } from 'prop-types';
+import styled, { css } from 'styled-components';
+import { C_EBON, C_WHITE, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING, GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import { GEL_MINION } from '@bbc/gel-foundations/typography';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const GEL_SPACING_DEC = '5rem';
+const BGC_TRANSITION_DURATION = '300ms';
+
+const Button = styled.button`
+  background-color: ${C_EBON};
+  border: none;
+  color: ${C_WHITE};
+  cursor: pointer;
+  display: block;
+  ${({ service }) => getSansBold(service)}
+  ${GEL_MINION};
+  height: ${GEL_SPACING_DEC};
+  padding: 0;
+  transition: background-color ${BGC_TRANSITION_DURATION};
+  width: ${GEL_SPACING_DEC};
+
+  &:hover,
+  &:focus {
+    background-color: ${C_POSTBOX};
+    transition: background-color ${BGC_TRANSITION_DURATION};
+  }
+`;
+
+const IconWrapper = styled.div`
+  > svg {
+    fill: ${C_WHITE};
```

Comment:
Can we make sure this works in a custom high contrast mode on PC and when the user changes colour preferences on FF please?